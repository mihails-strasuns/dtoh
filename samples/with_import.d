module withimport;

import ipath.defs;

extern(C) S foo();
