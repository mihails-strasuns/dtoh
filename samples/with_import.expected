#include <stdint.h>

// Used enum definitions:



// Struct forward declaration to resolve cycles:

struct S;

// Function pointer types:



// Used struct definitions:

struct S
{
    int32_t x;
};

// Variable and function declarations:

struct S foo();
